odata-url-conventions/5 Query Options.md

```diff
@@ -1928,14 +1928,17 @@ A property MUST NOT appear in more than one expand item.
 
 Query options can be applied to an expanded navigation property by
 appending a semicolon-separated list of query options, enclosed in
-parentheses, to the navigation property name. Allowed system query
-options are [`$filter`](#SystemQueryOptionfilter),
+parentheses, to the navigation property name. 
+Allowed system query options are
 [`$select`](#SystemQueryOptionselect),
+`$expand`, and
```

Comment:
Is this "and" helpful? Why not a plain comma-separated list?

Reply:
The first three are for all navigation properties (added now)